package.json

```diff
@@ -59,7 +59,7 @@
     "generate:readme:toc": "markdown-toc -i README.md",
     "generate:assets": "npm run build && npm run generate:readme:toc",
     "generate:template-parameters": "ts-node ./scripts/template-parameters.ts",
-    "generate:docker": "docker build --tag asyncapi/studio .",
+    "generate:docker": "docker buildx build --platform linux/amd64,linux/arm64 --tag asyncapi/studio .",
```

Comment:
I am not an expert of docker, so could you describe me this change? I mean, I have problem to understand how many (is it a good question) docker images will be created, with different architecture destination, or everything will be virtualised across one image? Thanks for explanation! I am afraid about backward compatibility.

Reply:
Of course! I'll try my best to explain.

"Docker Buildx is a CLI plugin that extends the docker command with the full support of the features provided by [Moby BuildKit](https://github.com/moby/buildkit) builder toolkit." [[1](https://docs.docker.com/build/buildx/)].

This is the way docker enables us to cross-compile docker images for different architectures. By using the `docker-container` Buildkit driver, the builds run inside a buildkit container, which may have support for QEMU emulation [[2](https://en.wikipedia.org/wiki/QEMU)] in order to compile for other architectures.

Other change that is a side effect is using docker Buildkit [[3](https://docs.docker.com/develop/develop-images/build_enhancements/)] as the build system, which was introduced in the 18.09 version and is currently the recommended build system to use. It also enables us to use new features in the Dockerfile, such as caching.

Reply:
@sandoval Ok, thank you very much! So tl;dr; it is a emulation 😄